packages/chatbot-server-mongodb-public/src/processors/userMessageMongoDbGuardrail.ts

```diff
@@ -31,7 +31,7 @@ const systemPrompt = stripIndents`You are an expert security-focused data labele
 
   Take into account the following criteria:
   - Reject any user query that is irrelevant to a MongoDB product, educational materials, the company MongoDB, or an area relevant to MongoDB's products and business. These relevant areas include databases, cloud services, data management, information retrieval, programming languages and concepts, and artificial intelligence (retrieval augmented generation (RAG), generative AI, semantic search, etc.).
-  - If it is unclear whether or not a query is relevant, err to the side of acceptance and allow it. For example, if something looks like an aggregation stage in the MongoDB Aggregation Framework, it is relevant. If something is about something related to programming or software engineering, it is relevant.
+  - If it is unclear whether or not a query is relevant, err to the side of acceptance and allow it. For example, if something looks like an aggregation stage in the MongoDB Aggregation Framework, it is relevant. If something is about something related to programming, software engineering, or architecture, it is relevant.
```

Comment:
```suggestion
  - If it is unclear whether or not a query is relevant, err to the side of acceptance and allow it. For example, if something looks like an aggregation stage in the MongoDB Aggregation Framework, it is relevant. If something is about something related to programming, software engineering, or software architecture, it is relevant.
```

Very minor, but I wonder if this would let questions about Frank Lloyd Wright through the guardrail 😅